students/arun_nalla/session06/test_Mailroom_Part4.py

```diff
@@ -0,0 +1,49 @@
+#!usr/bin/env python
+""" Testing Mailroom functions using pytest"""
+
+from Mailroom import get_report
+from Mailroom import donors_dict
+from Mailroom import donors_list
+from Mailroom import add_donors
+from Mailroom import email_message
+from Mailroom import path_to_store
+from Mailroom import add_donation
```

Comment:
Minor: You can combine all these imports into one line: `from Mailroom import get_report, donors_dict, donors_list, ...`. Also it's not that bad to use `from Mailroom import *` in a case like this (unit testing, that's very confined), even though we learned that `from ... import *` is not a good practice.